src/main/java/mate/academy/util/SaltUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class SaltUtil {
```

Comment:
This is not SaltUtil, this is HashUtil, because you make hash and not salt here. Also, maybe you need a method which will random generate the salt